plugins/package-managers/pub/src/funTest/assets/projects/external/dart-http-expected-output.yml

```diff
@@ -24,20 +24,20 @@ project:
       - id: "Pub:source_span:source_span:1.10.0"
         dependencies:
         - id: "Pub:collection:collection:1.18.0"
-        - id: "Pub:path:path:1.8.3"
+        - id: "Pub:path:path:1.9.0"
```

Comment:
Maybe say in the commit message that this is needed to make tests pass when run manually / that this was not necessary so far as tests were not run on CI.